### DISCUSSION TITLE
Python version boundation?

### DISCUSSION THREAD

**Discussion by Blank-c** [pyinstaller/pyinstaller#7006] [Help] [answered]
If I compile bootloader while having python 3.10, and then share the compiled pyinstaller to someone, do the "someone" need to have python 3.10 too to work??

**Comment by rokm** [comment, marked as answer]
No, the bootloader is not in any way tied to the API/ABI version of the python you use to compile the bootloader. 

Which is why we can provide a single set of pre-compiled bootloaders that can be used to create frozen applications using any supported python version.

**Comment by Blank-c** [comment]
Well thanks. Also is this related to pyinstaller? 
![IMG_20220731_205201](https://user-images.githubusercontent.com/94945186/182033762-7ad4642a-e2e3-45d9-abeb-d5ff9c302731.jpg)

**Reply by rokm** [reply]
This looks like a mix-up of python 3.8 and python 3.10 environment. I.e., you are running under python 3.10, but the search paths point to python 3.8 extensions (and possibly other modules). Either you have `PYTHONPATH` environment set incorrectly, or, if this error is raised while you try to run PyIntaller, you are setting search paths via `--paths` command-line arguments. I can't tell exactly which it is from this screenshot, though.

**Reply by Blank-c** [reply]
thanks
